### ISSUE TITLE
data-view-menu is not working in phonegap[solved]

### ISSUE THREAD

**Issue by chrcao** [tapquo/Lungo.js#261]
hi, I use menu with attribute data-view-menu, and it can work in firefox browser. but,
after I make a phonegap app and installed to my android device(version 4.2). nothing showed when I click menu button. could you help that. thanks.

**Comment by chrcao** [comment]
disappointed, no one response this problem.
but good news, I solved this issue. if anyone meets this issue again. set transition-duration to 0ms which is defined in lungo.css

**Comment by berthojoris** [comment]
Thanks for your trick,

Do you mean 
-webkit-transition-duration, -moz-transition-duration, -ms-transition-duration and -o-transition-duration
or just transition-duration ??

**Comment by chrcao** [comment]
yes, right. seems this css is not supported in phonegap.

**Comment by chrcao** [comment]
I set all values to 0, not just transition-duration

**Comment by berthojoris** [comment]
When you are sets, all goes smoothly in phonegap?

**Comment by chrcao** [comment]
yes, the result is ok for me.  <nav> block will show without any animation after set transition-duration to 0.

**Comment by berthojoris** [comment]
What about the lag issues?
I'm trying to create an application with lungo for phonegap, while still in the stages develop in the computer. Do you use the web application server to process the data?

**Comment by chrcao** [comment]
Do you mean lungo framework is not fast compared with native app? 
every mobile html framework has this issue. I think. because it is html, js, css...
current now. I have not try to load big data to html page. so I don't know the result. 
but I think workaround always can be found to improve it if it has terrible performance really.

**Comment by thakerhardiks** [comment]
Where's the solution? I am facing same problem ! :'(

**Comment by chrcao** [comment]
set transition-duration to 0ms which is defined in lungo.css

**Comment by thakerhardiks** [comment]
Not working.. Changed all transition-duration:0ms but not working !!

**Comment by chrcao** [comment]
maybe it is not a same problem.

**Comment by lordaugustus** [comment]
I notice strange lag on section transition on phonegap too. Any ideas?

**Comment by kutub-utara** [comment]
its work, thanks
